README.md

```diff
@@ -99,8 +95,34 @@ If you want to delete a collection of entries based on asearch, you can pass an
 db.delete({ job: dev })
 ```
 
-## To do
+### Dropping all entries
+
+To remove all existing entries, you can drop the database which basically resets everything to its initial state:
+
+```js
+db.drop()
+```
+
+### Counting existing entries
 
-- [x] Improve the way I deal with `drop`
-- [ ] Add `limit`, `sort` and other cool operations.
-- [ ] I'd like to do something like MongoDB for the update. Not great for now.
+You can use the `count(..)` method to count the number of existing entries:
+
+```js
+db.count()
+```
+
+## Development
+
+```
+# Installing dependencies
+npm install
+
+# Linting the code
+npm run lint
+
+# Running the tests
+npm run test --silent
```

Comment:
`npm test` is running the `test` script, as `npm start` runs the `start` script. Other "custon" scripts like `build` or `lint` don't have a global alias, so you have to explicitely `npm run` them.

Also `--silent` is okay, that was not my point!